backend/db/resolve_fetch_failures_0_members.py

```diff
@@ -20,8 +20,6 @@ def resolve_fetch_failures_0_members(version_id: int = None, use_local_db=False)
     :param version_id: Optional concept set version ID to resolve. If not provided, will check database for flagged
     failures."""
     print('Resolving fetch failures: new concept set / 0 members')
-    print('id:')
-    print(version_id)
     if version_id:
         print(f'Fetching version ID: {version_id}')
         with get_db_connection(local=use_local_db) as con:
```

Comment:
The following things were done in a commit directly made to `develop`, so I'm linking to those comments here.

## Set up of GitHub action
https://github.com/jhu-bids/TermHub/commit/29e8bcbe83d71f03e601e16a36597a4bff4a9353#r122809428

## GitHub action `version_id` param
https://github.com/jhu-bids/TermHub/commit/29e8bcbe83d71f03e601e16a36597a4bff4a9353#r122809466

## `call_github_action()` updates
https://github.com/jhu-bids/TermHub/commit/29e8bcbe83d71f03e601e16a36597a4bff4a9353#r122809553

## New: `call_github_action_with_params()`
https://github.com/jhu-bids/TermHub/commit/29e8bcbe83d71f03e601e16a36597a4bff4a9353#r122809481